clients/payment/src/components/Preview.tsx

```diff
@@ -0,0 +1,317 @@
+  sepaDirectDebit: string | undefined;
+  merchantName: string | undefined;
+};
+
+export const Preview = ({
+  accentColor,
+  logo,
+  merchantName,
+  label,
+  amount,
+  currency = "€",
+  card,
+  sepaDirectDebit,
+}: Props) => {
+  const languageOptions = useMemo(() => {
+    return languages.map(country => ({
+      name: country.native,
+      value: country.id,
+    }));
+  }, []);
+
+  const hasNoPaymentMethods = sepaDirectDebit == null && card == null;
+
+  const paymentMethods = [
+    ...(sepaDirectDebit != null || hasNoPaymentMethods
+      ? [
+          {
+            id: "1",
+            name: "Direct Debit",
+            icon: <SepaLogo height={15} />,
+          },
+        ]
+      : []),
+    ...(card != null || hasNoPaymentMethods
+      ? [
+          {
+            id: "2",
+            name: "Card",
+            icon: <Icon name="payment-regular" size={24} />,
+            activeIcon: <Icon name="payment-filled" size={24} />,
+          },
+        ]
+      : []),
+  ];
+
+  const [selectedPaymentMethod, setSelectedPaymentMethod] = useState(paymentMethods[0]);
+
+  return (
+    <WithPartnerAccentColor color={accentColor ?? invariantColors.defaultAccentColor}>
+      <ResponsiveContainer breakpoint={breakpoints.large} style={styles.root}>
+        {({ large }) => (
+          <View style={styles.base}>
+            <ScrollView contentContainerStyle={styles.content}>
+              <>
+                <Box direction="row" alignItems="center">
+                  <LakeButton
+                    ariaLabel={t("common.cancel")}
+                    icon="dismiss-regular"
+                    mode="tertiary"
+                    onPress={() => {}}
+                  >
+                    {large ? t("common.cancel") : null}
+                  </LakeButton>
+
+                  <Fill minWidth={16} />
+
+                  <View>
+                    <LakeSelect
+                      value={locale.language}
+                      items={languageOptions}
+                      hideErrors={true}
+                      mode="borderless"
+                      onValueChange={() => {}}
+                    />
+                  </View>
+                </Box>
+
+                <Space height={24} />
+              </>
+              {isNotNullish(logo) ? (
+                <AutoWidthImage
+                  height={40}
+                  maxWidth={180}
+                  resizeMode="contain"
+                  sourceUri={logo}
+                  style={styles.centered}
+                />
+              ) : (
+                <LakeHeading variant="h3" level={3} align="center">
+                  {merchantName}
+                </LakeHeading>
+              )}
+
+              <Space height={24} />
+
+              <LakeText variant="medium" align="center" color={colors.gray[700]}>
+                {label}
+              </LakeText>
+
+              <Space height={12} />
+
+              {amount != null ? (
```

Comment:
nit: just perform a condition on the content, as the Heading component keep the same props